jumpscale/packages/vdc_dashboard/chats/taiga.py

```diff
@@ -7,9 +7,11 @@ class TaigaDeploy(SolutionsChatflowDeploy):
     title = "Taiga"
     HELM_REPO_NAME = "marketplace"
     steps = ["get_release_name", "create_subdomain", "set_config", "install_chart", "initializing", "success"]
-    ADDITIONAL_QUERIES = [
-        {"cpu": 250, "memory": 256},  # postgers
-    ]
+    CHART_LIMITS = {
+        "Silver": {"cpu": "3000m", "memory": "3024Mi"},
+        "Gold": {"cpu": "4000m", "memory": "4096Mi"},
+        "Platinum": {"cpu": "5000m", "memory": "5120Mi"},
```

Comment:
the resources seems too large, do we really need it ?
if so we should enable it on certain plans only

Reply:
Yes they are large because the new taiga has 9 deployments. We can look into enabling it on certain plans :+1: